### PR TITLE
[IMP] account: Added feature to create recurrent vendor bills and jou…

### PR DESCRIPTION
**Pull request by rgo-odoo**
…rnal entries,

    - One need to first enable this from Accounting Settings to use this feature
    - Once activated, user can see one option in Vendor Bills and Journal Entries to set perticular record to be recurrent document and can define the recurrency period. This will automatically create draft documents(by mean of Scheduled Actions) as per recurrency configuration.

Task: https://www.odoo.com/web?debug=#id=34884&view_type=form&model=project.task&action=327&menu_id=4720
Pad: https://pad.odoo.com/p/r.2ce5d6bcc2c17bde4a157bc33163f7dc

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why only recurring vendor bills and not customer ones?

**Comment by rgo-odoo** [comment]
@pedrobaeza because customer ones are handled by sale_subscription app

**Comment by rgo-odoo** [comment]
LGTM

File: addons/account/data/ir_cron_data.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <!-- Cron job for creating recurring vendor bills -->
+    <record id="recurrent_vendor_bills_cron" model="ir.cron">
+        <field name="name">Account: generate recurring vendor bills</field>
+        <field name="model_id" ref="account.model_account_invoice"/>
+        <field name="state">code</field>
+        <field name="active">False</field>
+        <field name="code">model._cron_generate_recurring_vendor_bill()</field>
+        <field name="interval_number">1</field>
```
Comment (danger89) [inline addons/account/data/ir_cron_data.xml]: Can we make it possible to change the interval time in the settings?

**Pull request closed without merging**